app/src/sharedTest/java/org/oppia/app/walkthrough/WalkthroughActivityTest.kt

```diff
@@ -72,6 +72,7 @@ class WalkthroughActivityTest {
     }
   }
 
+  /* ktlint-disable max-line-length */
```

Comment:
Can we have kDoc for this somewhere in the class?

Reply:
this I will update and let you know in next commit

Reply:
Did you remove this line from this PR?

Reply:
yes, I bring these in a much cleaner way which is one line comment rather than a blocky comment.

Reply:
Okay, I think on top of the class if you mention that would be fine.

Reply:
I will be updating on the wiki as it will be on most of the places

Reply:
That would be good.